test/IDE/newtype.swift

```diff
@@ -50,6 +53,18 @@
 // PRINT-NEXT:    static let version: MyFloat
 // PRINT-NEXT:  }
 //
+// PRINT-LABEL: struct MyInt : RawRepresentable, _SwiftNewtypeWrapper, Equatable, Hashable, Comparable {
+// PRINT-NEXT:    init(_ rawValue: Int32)
+// PRINT-NEXT:    init(rawValue: Int32)
+// PRINT-NEXT:    let rawValue: Int32
+// PRINT-NEXT:  }
+// PRINT-NEXT:  extension MyInt {
+// PRINT-NEXT:    static let zero: MyInt!
+// PRINT-NEXT:    static let one: MyInt!
+// PRINT-NEXT:  }
```

Comment:
@jrose-apple @DougGregor 
Umm, in hindsight, why are zero and one "MyInt!", i.e., where is the optional? Am I missing something in the importer here?